domains/proplays.json

```diff
@@ -0,0 +1,10 @@
+{
+        "owner": {
+           "username": "ProPlays2312",
+           "email": "proplays2312@hotmail.com"
+        },
+  "record": {
+    "NS": "amit.ns.cloudflare.com",
```

Comment:
NS records are not supported per guidelines.

Reply:
My discord is proplays2312. Lets talk a bit there about this

Reply:
We literally don't support NS records because of our DNS provider. Don't know how to make that clear if you still don't know.